client/src/redux/project/project.types.ts

```diff
@@ -1,15 +1,21 @@
 export interface Project {
-	_id: string;
+	_id?: string;
 	title: string;
 	description: string;
-	image: string;
+	image?: string;
 }
 
 export const SET_ALL_PROJECTS = 'SET_ALL_PROJECTS';
-export const CREATE_PROJECT = 'CREATE_PROJECT';
+export const CREATE_PROJECT_START = 'CREATE_PROJECT_START';
+export const CREATE_PROJECT_SUCCESS = 'CREATE_PROJECT_SUCCESS';
 
-interface CreateProjectAction {
-	type: typeof CREATE_PROJECT;
+interface CreateProjectStartAction {
+	type: typeof CREATE_PROJECT_START;
+	payload: Project;
+}
+
+interface CreateProjectSuccessAction {
+	type: typeof CREATE_PROJECT_SUCCESS;
 	payload: Project;
 }
```

Comment:
The fundamental difference between these 2 actions is that the start action sends out details asking to create a project. Success is going to have the same project details but with an id attribute